.github/workflows/build-test-e2e.yml

```diff
@@ -66,10 +82,23 @@ jobs:
       - name: Run Playwright tests
         env:
           ADMIN_PAT: ${{ secrets.ADMIN_PAT }}
-        run: yarn playwright test --shard ${{ matrix.shard }}
+          TRACE_TOGGLE: ${{ github.repository_owner == 'Sage-Bionetworks' && 'on' || 'off'}}
+        run: yarn playwright test --shard ${{ matrix.shard }} --trace=${{ env.TRACE_TOGGLE }}
+      - name: Assume AWS Role
+        if: github.repository_owner == 'Sage-Bionetworks'
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-region: us-east-1
+          role-to-assume: arn:aws:iam::449435941126:role/sagebase-github-oidc-synapsedev-e2e-infra
+          role-session-name: GitHubActions-${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}
+          role-duration-seconds: 1200
+      - name: Upload blob report to S3
+        if: github.repository_owner == 'Sage-Bionetworks'
+        run: |
+          aws s3 sync ./blob-report --region us-east-1 s3://${{ env.DESTINATION_BUCKET }}/${{ env.REPORT_ID }}/
```

Comment:
Would a retention policy be specified here? Or is it specified on the bucket?

Reply:
The policy is specified on the bucket -- objects are set to expire after 14 days, which matches the retention set for reports stored as GitHub artifacts.